### PR TITLE
Scan agent changes to disable OSS CA telemetry - based on automation account variable

### PR DESCRIPTION
**Pull request by Aboli-msft**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

**Review by TarunKrShukla — APPROVED** [review]

**Review by ganesh-msft — APPROVED** [review]

File: src/AzSK/Framework/Configurations/ContinuousAssurance/RunbookScanAgent.ps1 [hunk 1]
```diff
@@ -84,9 +84,6 @@ function RunAzSKScan() {
 		Set-AzSKUsageTelemetryLevel -Level $usageTelelmetryLevel.Value.Trim()
```
Comment (mprabhu11) [inline src/AzSK/Framework/Configurations/ContinuousAssurance/RunbookScanAgent.ps1:84]: We should do Trim() at the innermost place where we copy from user input to internal setting

File: src/OSSConfiguration/RunbookScanAgent.ps1 [hunk 2]
```diff
@@ -71,7 +71,20 @@ function RunAzSKScan() {
...
+	Set-AzSKPrivacyNoticeResponse -AcceptPrivacyNotice "yes"
+	
+	# Check if automation acccount variable UsageTelemetryLevel is set 
+	# Set telemetry level as value of this variable
+	$usageTelelmetryLevel = Get-AzAutomationVariable -Name "UsageTelemetryLevel" -AutomationAccountName $AutomationAccountName -ResourceGroupName $AutomationAccountRG -ErrorAction SilentlyContinue
+
+	if($null -ne $usageTelelmetryLevel -and [string]::IsNullOrWhiteSpace($usageTelelmetryLevel.Value))
+	{
+		Write-Output("SA: Setting Telemetry usage to $($usageTelelmetryLevel.Value)")
+		Set-AzSKUsageTelemetryLevel -Level $usageTelelmetryLevel.Value.Trim()
```
Comment (mprabhu11) [inline src/OSSConfiguration/RunbookScanAgent.ps1:83]: same comment about Trim().

File: src/OSSConfiguration/RunbookScanAgent.ps1 [hunk 3]
```diff
@@ -71,7 +71,20 @@ function RunAzSKScan() {
...
+	}
+
+	#------------------------------------Clear session state to ensure updated policy settings are used-------------------
+	Clear-AzSKSessionState
```
Comment (mprabhu11) [inline src/OSSConfiguration/RunbookScanAgent.ps1:87]: CSS is to be done by Set-AzSKUsageTelemetryLevel

File: src/AzSK/Framework/Configurations/ContinuousAssurance/RunbookScanAgent.ps1 [hunk 4]
```diff
@@ -84,9 +84,6 @@ function RunAzSKScan() {
...
 	}
 
-	#------------------------------------Clear session state to ensure updated policy settings are used-------------------
-	Clear-AzSKSessionState
```
Comment (mprabhu11) [inline src/AzSK/Framework/Configurations/ContinuousAssurance/RunbookScanAgent.ps1]: unclear why this css shows as 'diff'.

**Pull request merged**